controller/src/localizations/en.ts

```diff
@@ -0,0 +1,6 @@
+const en = {
+  tw: "Translations Words",
+  tn: "Translation Notes",
+};
+export type keysType = keyof typeof en;
+export default {dict: en, ietf: "en"};
```

Comment:
For now, I know that you mentioned doing this in crowdin, and we could certainly do it via api most likely. For now though, I've put these into TS files.  We probably need to decide what's the scope of these to translate, and moreover, there's some junky data currently in content resource types that needs cleaning up where the resource type is clearly not somethign we would consider a resource type.  Probably worth having a discussion on .